### PR TITLE
1174: Add case contact notes

### PR DESCRIPTION
**Pull request by ahernp**
Trello card [#1174](https://trello.com/c/mO4YBD9m/1174-add-generic-notes-field-for-contact-details-design-attached).

### CODE REVIEW

File: accessibility_monitoring_platform/apps/audits/forms.py [hunk 1]
```diff
@@ -326,7 +326,6 @@ class AuditStatement1UpdateForm(VersionForm):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields["add_contact_email"] = AMPCharFieldWide(label="Email")
-        self.fields["add_contact_notes"] = AMPTextField(label="Notes")
```
Comment (richardbored) [inline accessibility_monitoring_platform/apps/audits/forms.py:329]: Is it possible to keep this field and the data in the database?

**Review by richardbored — CHANGES_REQUESTED** [review]
PR looks good. The only request can we not hard delete the notes data from the database?

File: accessibility_monitoring_platform/apps/audits/forms.py (hunk 1 above)
Comment (ahernp) [reply accessibility_monitoring_platform/apps/audits/forms.py:329]: I don't know how that would work.

Where would notes entered here be stored? If in `Case.contact_notes` then would existing content be shown here for the user to edit? If not then the users would be blindly entering notes which get appended to existing text.

**Comment by ahernp** [comment]
> PR looks good. The only request I have is can we not hard delete the notes data from the database?

The PR migrates contents from `Contact.notes` to `Case.contact_notes`. Do you mean that you want the migration to still happen but that the source data be left in place and not shown in the UI?

**Comment by richardbored** [comment]
> > PR looks good. The only request I have is can we not hard delete the notes data from the database?
> 
> The PR migrates contents from `Contact.notes` to `Case.contact_notes`. Do you mean that you want the migration to still happen but that the source data be left in place and not shown in the UI?

Yes essentially. If this isn't possible, I'm happy to push this through as is.

**Review by richardbored — APPROVED** [review]

**Pull request merged**